doc/source/security/encrypting-binlogs.rst

```diff
@@ -0,0 +1,77 @@
+.. _encrypting-binlogs:
+
+=======================================================================
+Encrypting Binary Logs
+=======================================================================
+
+Binary log encryption at rest ensures the server-generated binary logs are encrypted in persistent storage.
```

Comment:
Please also use a consistent formatting accross files. E.g. the double write file above and some others use 80 character line breaks, while this file and some others only break lines after a sentence ends.

Reply:
Updated.